app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Ai grija in acest layout ca marginRight si marginLeft sa fie corelate cu marginEnd si marginStart. De asemenea, gravity right ar trebui sa fie inlocuit cu gravity end. Asta e pentru a acomoda utilizatorii care citest rightToLeft.